clients/spark-extensions/src/test/java/org/projectnessie/spark/extensions/ITNessieStatements.java

```diff
@@ -60,6 +62,7 @@ protected static void createDelta() {
 
   @BeforeEach
   void getHash() throws NessieNotFoundException {
+    nessieClient = NessieClient.builder().withUri(url).build();
```

Comment:
why not also use the same Nessie API as in the other classes here?

Reply:
It's using "old Nessie" here